### ISSUE TITLE
production.ERROR: htmlspecialchars() expects parameter 1 to be string, array given

### ISSUE THREAD

**Issue by getinfo-developer** [Attendize/Attendize#469]
Hi , Getting following error when clicking on REFERENCE from Orders. It looks like issue in encoding email id

production.ERROR: htmlspecialchars() expects parameter 1 to be string, array given (View: /Users/vip/Sites/Attendize/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/views/500.blade.php) {"userId":2,"email":"getvip@gmail.com","exception":"[object] (ErrorException(code: 0): htmlspecialchars() expects parameter 1 to be string, array given (View: /Users/vip/Sites/Attendize/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/views/500.blade.php) at /Users/vip/Sites/Attendize/vendor/laravel/framework/src/Illuminate/Support/helpers.php:560, ErrorException(code: 0): htmlspecialchars() expects parameter 1 to be string, array given at /Users/vip/Sites/Attendize/vendor/laravel/framework/src/Illuminate/Support/helpers.php:560)
[stacktrace]

**Comment by jeremyquinton** [comment]
@getinfo-developer any idea on how I can replicate this issue. Some steps to replicate it will help me fix. 

Did you get this on the master branch I haven't been able to replicate it but will label it as a bug for now.

**Comment by venturism** [comment]
When a 500 error occurs an array is being passed to the page template triggering the htmlspecialcars() error.

`array (
  'back_soon' => 'We\'ll be back soon',
  'back_soon_description' => 'We\'re currently making improvements to our website.',
)`

**Comment by jeremyquinton** [comment]
@venturism thanks for for reporting will replicate and then fix thanks.

**Comment by venturism** [comment]
Quick follow up, I've now seen the the 500 error page which uses the above message array.

The original error (htmlspecialchars) is reported when there is a lower level error, such as a bad .env setting so the error is passed to the default 500.blade.php file in Laravel. 

If you break things tinkering with the app then you'll see the friendly 'we're making improvements' message.

**Comment by jeremyquinton** [comment]
@venturism appreciate your input on this. 

To replicate the issue in my development environment I stopped the database container which generates the 500 error. I was then able to get the error. 

```production.ERROR: htmlspecialchars() expects parameter 1 to be string, array given```

The issue is that when there is a 500 error its trying to use the default Laravel 500 error handler view. I added a customer error view. I'll commit a fix for this in the coming days.
